packages/components/src/components/accordion/accordion.jsx

```diff
@@ -4,15 +4,20 @@ import React from 'react';
 import { usePrevious } from '../../hooks';
 import Icon from '../icon';
 
-const Accordion = ({ className, icon_close, icon_open, list }) => {
+const Accordion = ({ className, icon_close, icon_open, list, is_expand_all }) => {
     const [open_idx, setOpenIdx] = React.useState(null);
+    const [expand_all, setExpandAll] = React.useState(is_expand_all);
 
     const prev_list = usePrevious(list);
 
     React.useEffect(() => {
         if (prev_list !== list) setOpenIdx(null);
     }, [list, prev_list]);
 
+    React.useEffect(() => {
+        setExpandAll(is_expand_all);
+    }, [is_expand_all]);
```

Comment:
```suggestion
    React.useEffect(() => {
        setExpandAll(is_expand_all);
    }, [is_expand_all]);
```

By right, you don't need this effect. You can directly use `is_expand_all`

Reply:
Will remove the useEffect